app/views/users/index.html.erb

```diff
@@ -1 +1,8 @@
-<h2>Home page</h2>
+<h1> List all users </h1>
+<div>
+   <ul class='list'>
+      <% @users.each do |user| %>
+         <%= render 'shared/user', user: user %>
```

Comment:
[OPTIONAL] I think you should add a `li` tag here since. you have a `ul` wrapping it